mitodl/mit-open
# ignore formatting of yarn folder

# What are the relevant tickets?
None

# Description (What does it do?)
This issue adds the vendored[^1] yarn binary `.yarn/` folder to list of folders `prettier` should ignore when checking formatting.

[^1]: For better or worse, the `.yarn/` folder is committed to source control (see [their faq](https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored) and [github issue](https://github.com/yarnpkg/berry/issues/969). 

Since `.yarn/` is a vendored folder, we don't care about its formatting. Ignoring its formatting makes Renovate work better with `pre-commit`. Currently, there's a weird interaction between Renovate and pre-commit:

```
[@renovate]     Update Yarn to v3.6.4                    X [83248b1]
[pre-commit.ci] auto fixes from pre-commit.com hooks     ✓ [d8a524e]

> Renovate will not automatically rebase this PR, because it does not recognize the last commit author and assumes somebody else may have edited the PR.
```


# How can this be tested?
If you really want to test this:
1. Check out https://github.com/mitodl/mit-open/pull/156/commits/75d57a9747d5f3d75d58c34e9d1c361fc9239064, on which `pre-commit` is failing.
2. Run `pre-commit run --all` to see that it fails.
3. Merge this branch in, and check that `pre-commit` no longer fails.
